xml/deployment_sysreqs.xml

```diff
@@ -47,6 +47,67 @@
    size. For the current maximum supported number of nodes, please refer to
    the Release Notes on <link xlink:href="https://www.suse.com/releasenotes/"/>.
   </para>
+
+  <sect2 xml:id="sec.deploy.requirements.system.cluster.salt_cluster_size">
+   <title>Salt Cluster Sizing</title>
+   <para>
+    &productname; relies on SaltStack (or Salt for short) to automate various
+    tasks. The actions are performed by so called "workers". A worker can handle
```

Comment:
Hmmm... Salt's architecture in our product has the salt-master and salt-minions. The salt-minions are the agents that run different operations on all machines, and the salt-master has this notion of workers, that is the number of channels it has open in order to send commands and receive their result from the salt-minions.

I'm not sure if saying that "actions are performed by so called 'workers'" is accurate. Workers is just the concept salt-master has for how many concurrent channels can be open at any given time, that will connect to salt-minions.